aspen/src/lib.rs

```diff
@@ -71,33 +71,40 @@ impl Linter {
             .transpose()?;
 
         let injected_trees = self
-            .injection
-            .as_ref()
-            .map(|Injection { query, language }| {
-                // run the injection query with the injection query and the original language
+            .injections
+            .iter()
+            .filter_map(|Injection { query, language }| {
```

Comment:
is this how one destructures a struct in the arg list?

Reply:
yup, is it ugly?

Reply:
idk honestly, I have not kept up with convenience syntax for some time